tests/backend_test.py

```diff
@@ -1017,7 +1016,7 @@ def test_compilation_correctness(perth_backend: IBMQBackend) -> None:
     c.CX(0, 3).CX(0, 4)
     u_backend = AerUnitaryBackend()
     u = u_backend.run_circuit(c).get_unitary()
-    ibm_backend = perth_backend
+    ibm_backend = AerUnitaryBackend()
```

Comment:
The unitary calculation only works for 14 or less entry's in the coupling map. There are no devices of that size available...

Reply:
Isn't this test now useless, since it's just comparing `AerUnitaryBackend()` with itself? (Also it doesn't need the backend parameter and doesn't need to be a remote-only test.) Can we solve the problem by removing blank wires in the compiled circuit?

Reply:
remove_blank_wires is still not working, had to use `FlattenRelabelRegistersPass` I will now keep the remote backend as it was before?

Reply:
(done in [25c7929](https://github.com/CQCL/pytket-qiskit/pull/220/commits/25c7929176171c358ad2b9d5904cb9da83a308db) )